### DISCUSSION TITLE
any plans to support mssql?

### DISCUSSION THREAD

**Discussion by yuphing-ong** [go-jet/jet#298] [Q&A]
The drivers from MS, at  "github.com/microsoft/go-mssqldb" (for sql server) and  "github.com/microsoft/go-mssqldb/azuread" (for azure) work with database/sql.  Just wondering if there's any work to have this supported in go-jet as well, or is this outside of scope/roadmap?

**Comment by go-jet** [comment]
Not in the near term.

**Reply by mispp** [reply]
has situation regarding this changed?

**Reply by go-jet** [reply]
No, since mssqldb is closed source(and from microsoft) it has the lowest priority.
